back/spec/models/idea_spec.rb

```diff
@@ -233,11 +232,13 @@
         end
 
         context 'when the idea belongs to an ideation phase, and the current phase is information' do
-          let(:project) { create(:project_with_past_ideation_and_current_information_phase, input_term: 'issue') }
+          let(:project) { create(:project_with_past_ideation_and_current_information_phase) }
           let(:ideation_phase) { project.phases.first }
+          let(:information_phase) { project.phases.last }
           let(:idea) { build(:idea, project: project, phases: [ideation_phase]) }
 
           it 'returns the input_term of the ideation phase' do
+            information_phase.update!(input_term: 'issue')
             ideation_phase.update!(input_term: 'question')
             expect(idea.input_term).to eq 'question'
```

Comment:
Perhaps a reload needs to happen here to ensure that the phase update is taken into account?